orchestrator/services/processes.py

```diff
@@ -222,8 +221,6 @@ def _db_log_process_ex(pid: UUID, ex: Exception) -> None:
 
 
 def _run_process_async(pid: UUID, f: Callable) -> Tuple[UUID, Future]:
-    debugger = app_settings.POST_MORTEM_DEBUGGER
```

Comment:
the SURF specific repo has a paragraph in the README about post mortem debugger: do we need to remove that as well?

Reply:
Already did :-)